include/d/actor/d_a_player.h

```diff
@@ -109,7 +112,7 @@ class daPy_py_c : public fopAc_ac_c {
     virtual void checkCutCharge() const;
     virtual void getBokoFlamePos(cXyz*);
     virtual void checkTactWait() const;
-    virtual void setTactZev(unsigned int, int, char*);
+    virtual void setTactZev(unsigned int, int, const char*);
```

Comment:
The debug maps show this param isn't const, changing it to be const might break regalloc